src/AbstractActionReader.ts

```diff
@@ -57,68 +63,49 @@ export abstract class AbstractActionReader {
    * `nextBlock`.
    */
   public async getNextBlock(): Promise<NextBlock> {
-    let blockData = null
-    let isRollback = false
-    let isNewBlock = false
+    const blockMeta = {
```

Comment:
Same question here

Reply:
This is no longer a property on the class, because it doesn't need to be.